rune/libenclave/attestation/sgx/ias/ias.go

```diff
@@ -204,20 +204,34 @@ func (ias *iasService) Verify(quote []byte) *attest.Status {
 	var err error
 	if resp, err = ias.reportAttestationEvidence(p); err != nil {
 		status.ErrorMessage = fmt.Sprintf("%s", err)
-		return status
+		return status, nil, err
 	}
 	defer resp.Body.Close()
 
 	var reportStatus *reportStatus
-	if reportStatus, err = checkVerificationReport(resp, quote, nonce); err != nil {
+	reportStatus, iasReport, err := checkVerificationReport(resp, quote, nonce)
+	if err != nil {
 		status.ErrorMessage = fmt.Sprintf("%s", err)
-		return status
+		return status, nil, err
 	}
 
```

Comment:
formatIasReport() should be called here, rather than being called inside checkVerificationReport().